tests/testTriggers.go

```diff
@@ -2750,6 +2758,8 @@ func TriggerVolumeClone(contexts *[]*scheduler.Context, recordChan *chan *EventR
 					log.InfoD(stepLog)
 					log.Infof("Calling CloneVolume()...")
 					clonedVolID, err = Inst().V.CloneVolume(vol.ID)
+					log.InfoD("Printing the volume inspect for the volume:%s ,volID:%s and namespace:%s", vol.Name, vol.ID, vol.Namespace)
```

Comment:
remove the log.infoD or change it to Debugf
Also print the volume details after source node event also

Reply:
changed it from infod to debugf and also added the volumeinspect after sourcenode event